pom.xml

```diff
@@ -902,6 +902,71 @@
 
 			</properties>
 		</profile>
+		        
+		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
+        <profile>
+			<id>sigmah4hi</id>
```

Comment:
@osarrat yes you're right, NGO is not an option. I'm not ok with `userbuild` because `user` may refer to pretty anybody. I'm ashamed I didn't thought about `custom` before since it's seem quite obvious.

Is "sigmah-custom-*" ok for you?

You just have to commit on your local repo (master branch) and push it to your own github fork. The current pull-request should consider your change. You may also fix the indentation which seem broken (line 907).

I'm sorry this discussion takes time but naming things and making things unambiguous for everyone, including new developers, is very important. Also, this is a nightmare to changer later so i prefer take everyone two minutes now than having support to do later.